apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -138,9 +138,9 @@ type TLS struct {
 
 // IngressConfig defines ingress specific config items.
 type IngressConfig struct {
-	// Ingress Class Name Contour should use.
+	// Ingress Class Names Contour should use.
 	// +optional
-	ClassName *string `json:"className,omitempty"`
+	ClassName []string `json:"className,omitempty"`
```

Comment:
May as well update the names while the API is alpha:
```suggestion
	ClassNames []string `json:"classNames,omitempty"`
```